<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>809</width>
    <height>575</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>qtLogger</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>335</y>
      <width>761</width>
      <height>211</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout" columnstretch="2,0,0,0,0,0,0">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="horizontalSpacing">
      <number>35</number>
     </property>
     <property name="verticalSpacing">
      <number>13</number>
     </property>
     <item row="0" column="1">
      <widget class="QPushButton" name="addEntry">
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Ubuntu</family>
        </font>
       </property>
       <property name="text">
        <string>Add new</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="editEntry">
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Edit</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="deleteEntry">
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_7">
       <property name="font">
        <font>
         <family>Ubuntu</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Name</string>
       </property>
      </widget>
     </item>
     <item row="4" column="3">
      <widget class="QLineEdit" name="frequency"/>
     </item>
     <item row="3" column="3">
      <widget class="QDateEdit" name="dateEdit"/>
     </item>
     <item row="2" column="2">
      <widget class="QLabel" name="label_4">
       <property name="font">
        <font>
         <family>Ubuntu</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>UTC</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <family>Ubuntu</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Country</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>Ubuntu</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Call</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="name"/>
     </item>
     <item row="4" column="1">
      <widget class="QComboBox" name="country"/>
     </item>
     <item row="2" column="3">
      <widget class="QTimeEdit" name="timeEdit"/>
     </item>
     <item row="0" column="6">
      <widget class="QPushButton" name="openQRZDatabase">
       <property name="font">
        <font>
         <family>Ubuntu</family>
        </font>
       </property>
       <property name="text">
        <string>Find in QRZ</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="call"/>
     </item>
     <item row="4" column="2">
      <widget class="QLabel" name="label_5">
       <property name="font">
        <font>
         <family>Ubuntu</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Frequency</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <family>Ubuntu</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Date</string>
       </property>
      </widget>
     </item>
     <item row="4" column="4">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>kHz</string>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item row="3" column="4">
      <widget class="QLabel" name="label_6">
       <property name="font">
        <font>
         <family>Ubuntu</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Band</string>
       </property>
      </widget>
     </item>
     <item row="3" column="6">
      <widget class="QComboBox" name="band"/>
     </item>
     <item row="2" column="6">
      <widget class="QCheckBox" name="qsl">
       <property name="font">
        <font>
         <family>Ubuntu</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>QSL?</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QPushButton" name="exportEntry">
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Export</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>761</width>
      <height>291</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>759</width>
       <height>289</height>
      </rect>
     </property>
     <widget class="QTableView" name="tableView">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>761</width>
        <height>291</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>tableView</sender>
   <signal>clicked(QModelIndex)</signal>
   <receiver>MainWindow</receiver>
   <slot>tableViewClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>111</x>
     <y>293</y>
    </hint>
    <hint type="destinationlabel">
     <x>31</x>
     <y>358</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>tableViewClicked()</slot>
 </slots>
</ui>
